### ISSUE TITLE
Where to start to train?

### ISSUE THREAD

**Issue by sunnyHelen** [YvanYin/VNL_Monocular_Depth_Prediction#2]
Sorry, I don't know if you loaded the train file and the function to calculate the loss.  I just didn't find them.

**Comment by YvanYin** [comment]
Hi, 
We didn't release the training codes. We will release them later.

**Comment by doantientai** [comment]
I am still looking forward for your training codes :D
